Adding testing scenarios (#2038)
microsoft/devhome: tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/AppInstall.md

# App Install Tests
If your code affects app install in Machine Configuration, please manually verify these scenarios.

## Scenarios
Please make sure to verify all these scenarios.

1. User can search for applications in search bar
1. User can install applications

> 1. User can not install an application that is already installed 

Reviewer: ```suggestion
1. User cannot install an application that is already installed 
```
